app/lib/task/cloudcompute/googlecloudcompute.dart

```diff
@@ -103,6 +105,13 @@ CloudCompute createGoogleCloudCompute({
       'must only contain [a-z0-9_-]',
     );
   }
+  if (maxRunDuration.isNegative) {
```

Comment:
Also check for null?

Reply:
It's not nullable, I could check for zero.